app/views/services/slovensko_sk_api/index.html.erb

```diff
@@ -34,7 +34,7 @@
           </div>
           <ul class="list-group">
             <li class="list-group-item"><strong>Open Source</strong></li>
-            <li class="list-group-item"><a href="https://generator.swagger.io/index.html?url=https://dev.slovensko-sk-api.staging.slovensko.digital/openapi.yaml" target="_blank">Základná funkcionalita</a></li>
```

Comment:
Linkujme na ten jeden swagger.

Reply:
Upravil som to tak, ze pod opensource je link "Zakladna funkcionalita" (free dokumentacie) a pod komercnym je link "Rozsirena funkcionalita" (premium dokumentacia), takto to nechceme?

Reply:
Podla mna nie, je to jasne zo swaggera a chceme, aby si to hned vsimli. Swagger dajme rovnaky.